packages/table/src/cell/formats/truncatedFormat.tsx

```diff
@@ -92,12 +95,24 @@ export class TruncatedFormat extends React.Component<ITruncatedFormatProps, ITru
         truncationSuffix: "...",
     };
 
-    public state: ITruncatedFormatState = { isTruncated: false };
+    public state: ITruncatedFormatState = {
+        isPopoverOpen: false,
+        isTruncated: false,
+    };
 
     private contentDiv: HTMLDivElement;
+    private cachedFontString: string;
```

Comment:
we cache the font string on first use

Reply:
this will be out of date if the style changes, but async font loading will not be an issue since the fontstring will still apply once the font is loaded.